packages/example/exercises/01.nested-routing/01.problem.outlet/package.json

```
@@ -21,11 +21,11 @@
     "validate": "run-p \"test -- --run\" lint typecheck"
   },
   "dependencies": {
-    "@remix-run/express": "^1.17.0",
-    "@remix-run/node": "^1.17.0",
-    "@remix-run/react": "^1.17.0",
-    "@remix-run/serve": "^1.17.0",
-    "@remix-run/server-runtime": "^1.17.0",
+    "@remix-run\/express": "^1.18.0",
+    "@remix-run\/node": "^1.18.0",
+    "@remix-run\/react": "^1.18.0",
+    "@remix-run\/serve": "^1.18.0",
+    "@remix-run\/server-runtime": "^1.18.0",
```

```suggestion
    "@remix-run/express": "^1.18.0",
    "@remix-run/node": "^1.18.0",
    "@remix-run/react": "^1.18.0",
    "@remix-run/serve": "^1.18.0",
    "@remix-run/server-runtime": "^1.18.0",
```

Pretty sure this is a mistake as well.